helix-core/src/main/java/org/apache/helix/manager/zk/ZkCacheBaseDataAccessor.java

```diff
@@ -263,7 +263,7 @@ public boolean set(String path, T data, int expectVersion, int options) {
       if (cache != null) {
         cache.lockWrite();
         ZkBaseDataAccessor<T>.AccessResult result =
-            _baseAccessor.doSet(serverPath, data, expectVersion, options);
+            _baseAccessor.doSet(_zkclient, serverPath, data, expectVersion, options);
```

Comment:
If we keep the old public API instead of removing, we shouldn't need to change this at all. In other words, we shouldn't see any diffs in CacheBaseDataAccessor.